sql-schema/127.sql

```diff
@@ -0,0 +1,11 @@
+DROP TABLE IF EXISTS `ipmi_sensors`;
```

Comment:
the component table can not store the information needed for me, unless i alter it (adding one more varchar field) that will hold the hardware id